### PR TITLE
#65/create landing UI page

### LINKED ISSUE #65: ⚛️ Create landing UI page

### PR DESCRIPTION
**Pull request by gbudau**
close #65

### CODE REVIEW

**Comment by Damrod** [comment]
For some reason that I cannot understand, I cannot see the mobile version on a browser. I have tried several times, and only once it has worked, by inspecting in chrome and using the device toolbar to see as if it was a google pixel 5 phone...

**Comment by Damrod** [comment]
how do you manage to make it navigate forward to the users page after pressing login?

**Review by fizcamposancos — APPROVED** [review]
I like it. Good work!

**Review by Damrod — APPROVED** [review]
Besides the comments, it looks great to me

**Comment by gbudau** [comment]
> For some reason that I cannot understand, I cannot see the mobile version on a browser. I have tried several times, and only once it has worked, by inspecting in chrome and using the device toolbar to see as if it was a google pixel 5 phone...

@Damrod
I think in Chrome is <kbd>Ctrl + Shift + I</kbd> to open the Inspect, then <kbd>Ctrl + Shift + M</kbd> to open the device toolbar.

Alternatively, we could use an app like [Responsively](https://responsively.app/) to view multiple devices.

File: webapp/src/pages/Landing/Landing.css [hunk 1]
```diff
@@ -1,13 +1,242 @@
...
+  max-width: 25rem;
+}
+
+/* Styles for viewports wider than 768 pixels. */
+@media (min-width: 768px) {
+  .landing {
+    min-height: 20rem;
+    gap: 6rem 1rem;
+    padding: 4rem 4rem 8rem 4rem;
+    grid-template-columns: repeat(12, 1fr);
```
[the remark this answers is not in the document]
Comment (gbudau) [reply webapp/src/pages/Landing/Landing.css]: Done :+1:

**Comment by gbudau** [comment]
Thank you! :tada:

**Pull request merged**